Updated the Default Routing: Execute Request page for v8

erikjanwestendorp opened the pull request:
https://github.com/umbraco/UmbracoDocs/issues/1873

- needsV8Update to false
- ContentModel instead of RenderModel
- Deleted DynamicPublishedContent

sofietoft commented:
This is great @erikjanwestendorp !! 😄 
Thanks for help getting it updated.

All examples and stuff looks great.
2 small things:
- Instead of changing the value to `false` you can also remove the "needsV8Update" - up to you!
- And then, on the article for Umbraco 7 - the links to the Querying section.. Would be great if you could update those, so they point to the Umbraco 7 version of those articles as well?
I believe they've all been versioned 💪

erikjanwestendorp commented:
@sofietoft Thanks for your quick reply on this.  In my latest commit I deleted "needsV8Update" and updated the links in the v7 article where possible. 

Updated links:

- IPublishedContent
- UmbracoHelper

No specific v7 links for DynamicPublishedContent and MembershipHelper as far as i know.

sofietoft commented:
Brilliant! 😄 

Yes, you're absolutely right.
Thanks for updating the links, and for putting in the work 💪😁 
I'll get this merged right away!
